### DISCUSSION TITLE
'sub' vs 'word' modifier

### DISCUSSION THREAD

**Discussion by MajorDallas** [cursorless-dev/cursorless#2224] [Q&A] [answered]
I wanted to select the last two parts of a token, `get_from_origin` (with r having the mark). I'm new to Cursorless so I keep the docs and cheatsheet open on one screen most of the time. In this case, I was looking at https://www.cursorless.org/docs/#sub-token-modifiers, which describes the `sub` modifier for this purpose. I tried `take second past last sub red`, but the subtitles would never quite match, sometimes substituting "state" for "sub" and often doing all kinds of weird things to my code.

Fortunately, [this comment](https://github.com/cursorless-dev/cursorless/issues/54#issuecomment-872061752) directed me to try `word` instead of `sub`, and that worked on the first try. Saved me a lot of frustration 🙂 .

I can't rule out that it's my microphone, which is a low-mid range gaming headset (which is why this is a Q&A and not an issue). However, I'm just curious if maybe the docs are out of date, and `sub` has been superseded by `word` since some time ago?

**Comment by pokey** [comment, marked as answer]
We used to use "word", but switched to "sub" in the past couple months. You must have installed Cursorless before we made that switch. Even if you update Cursorless, we preserve whatever terms you had when you installed, so that the grammar doesn't change from underneath you

Fwiw if you say "cursorless cheatsheet" you should see "word" in there
